gTea/app/src/main/java/com/nechaieva/gtea/MainActivity.java

```diff
@@ -48,6 +47,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
 
     public Fragment getVisibleFragment() {
+        // It is assumed that the first visible fragment that we find
+        // is that last one that is opened, since the fragment stack might
+        // not work in this case:
+        // the fragment stack initiates as empty.
+        // It works for the case when we simply navigate between fragments:
+        // then the account fragment, for which this part is made of,
+        // simply works as part of the navigation, just specified in code.
+        // Still, better long-term solution would be to somehow add it to
+        // nav_graph or update the back stack manually.
```

Comment:
Remark: for me this comment is difficult to comprehend. Consider reformulating (e.g. "This functions assumes xyz") and checking the assumptions in code. For now it is OK to leave it, as we discussed offline that fragment handling is not our priority at the moment.